dev-docs/examples/stored-impressions.md

```diff
@@ -0,0 +1,210 @@
+
+Every option has a certain use case. Choose the one that fits best.
+
+You can learn more about the general configuration here:
+
+- [adunit reference on stored impressions](/dev-docs/adunit-reference.html#stored-imp)
+- [prebidServer module for prebid.js](/dev-docs/modules/prebidServer.html)
+
+### Reusing client-side configuration
+
+{% capture tipNoteClientSide %}
+Use this if you want to AB test client vs server
+{% endcapture %}
+
+{% include alerts/alert_tip.html content=tipNoteClientSide %}
+
+{% capture warnNoteClientSide %}
+This requires client-side adapters to be present
+{% endcapture %}
+
+{% include alerts/alert_warning.html content=warnNoteClientSide %}
+
+```js
+pbjs.setConfig({
+    // configure a single prebid server instance
+    s2sConfig: [{
+        accountId: '1001',
+        bidders: ['bidderA'],
+        endpoint: 'https://mypbs.example.com/path',
+        syncEndpoint: 'https://mypbs.example.com/path',
+        timeout: 1000
+    }]
+});
+
+pbjs.addAdUnits({
+    code: "test-div",
+    bids: [
+        {
+            // this bidder is part of the bidders array, so it will not make client-side calls, but
+            // there params will be transfered to prebid server
+            bidder: 'bidderA',
+            params: {
+                placementId: 12345
+            }
+        },
+        {
+            // this one stays client-side
+            bidder: 'bidderB',
+            params: {
+                placementId: 55555
+            }
+        }
+    ]
+});
+```
+
+### adUnit.ortb2Imp configuration
+
+{% capture tipNoteOrtb2Imp %}
+When using [PBS stored impressions](/dev-docs/modules/prebidServer.html#stored-imp), the `AdUnit.bids` array is not required.
+Use this if you want to configure everything server side and have a **single** prebid server endpoint.
+{% endcapture %}
+
+{% include alerts/alert_tip.html content=tipNoteOrtb2Imp %}
+
+```js
+pbjs.setConfig({
+    // configure a single prebid server instance
+    s2sConfig: [{
+        accountId: '1001',
+        bidders: ['bidderA', 'bidderB'],
+        endpoint: 'https://mypbs.example.com/path',
+        syncEndpoint: 'https://mypbs.example.com/path',
+        timeout: 1000
+    }]
+});
+
+pbjs.addAdUnits({
+    code: "test-div",
+    ortb2Imp: {
+        ext: {
+            prebid: {
+                storedrequest: {
+                    id: 'stored-request-id'
+                }
+            }
+        }
+    }
+    // no bids object
+});
+```
+
+See [defining endpoints for prebid server](/dev-docs/modules/prebidServer.html#defining-endpoints)
+
+### bids[].module configuration
+
+{% capture tipNoteModule %}
+Use [PBS stored impressions](/dev-docs/modules/prebidServer.html#stored-imp) together with client-side bidders - or stored impressions from multiple instances of Prebid Server 
+{% endcapture %}
+
+{% include alerts/alert_tip.html content=tipNoteModule %}
+
+```js
+pbjs.setConfig({
+    s2sConfig: [
+      {
+          name: "pbs-appnexus",
```

Comment:
I've been trying to avoid naming real bidders in the documentation. Unless of course this example actually works to call a specific PBS and adapter?

Reply:
Nope, not really. You are about naming bidders ( even if the young folks may not remember who appnexus is 😅 )